extensions/amp-iframe/amp-iframe.md

```diff
@@ -16,7 +16,56 @@ limitations under the License.
 
 ### <a name="amp-iframe"></a> `amp-iframe`
 
-Displays an iframe.
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays an iframe.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/everything.amp.html">everything.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-iframe` tag
+(see also `amp-iframe` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
```

Comment:
optional, similar to above, there are several attribute values that have specific rules or will trigger an INVALID_ATTR_VALUE error